pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -294,3 +335,89 @@ func containsIdentityProvider(providers []*aerogearv1.KeycloakIdentityProvider,
 	}
 	return false
 }
+
+func getUserDiff(keycloakUsers []*aerogearv1.KeycloakUser, openshiftUsers []usersv1.User) ([]usersv1.User, []int) {
+	var added []usersv1.User
+	for _, osUser := range openshiftUsers {
+		if !kcContainsOsUser(keycloakUsers, osUser) {
+			added = append(added, osUser)
+		}
+	}
+
+	var deleted []int
+	for i, kcUser := range keycloakUsers {
+		if kcUser.UserName != CustomerAdminUser.UserName && !OsUserInKc(openshiftUsers, kcUser) {
+			deleted = append(deleted, i)
+		}
+	}
+
+	return added, deleted
+}
+
+func syncronizeUserLists(keycloakUsers []*aerogearv1.KeycloakUser, openshiftUsers []usersv1.User) []*aerogearv1.KeycloakUser {
+	added, deletedIndexes := getUserDiff(keycloakUsers, openshiftUsers)
+
+	var result []*aerogearv1.KeycloakUser
+	result = append(result, keycloakUsers...)
+
+	for _, index := range deletedIndexes {
+		result = remove(index, result)
+	}
+
+	for _, osUser := range added {
+		result = append(result, &aerogearv1.KeycloakUser{
+			KeycloakApiUser: &aerogearv1.KeycloakApiUser{
+				Enabled:       true,
+				Attributes:    aerogearv1.KeycloakAttributes{},
+				UserName:      osUser.Name,
+				EmailVerified: true,
+				Email:         osUser.Name + "@example.com",
+				ClientRoles: map[string][]string{
+					"account": {
+						"manage-account",
+						"view-profile",
+					},
+					"realm-management": {
+						"manage-users",
+						"manage-identity-providers",
+						"view-realm",
+					},
```

Comment:
I think that we don't need to give `realm-management` roles to any user now.
Customer-admin / dedicated-admins will be able to manage users in openshift or in the IDP that's configured in openshift. I think that was the only reason for giving them access.

Reply:
Ok. I'll update this :+1: